libs/wire-api/src/Wire/API/Asset.hs

```diff
@@ -177,8 +177,13 @@ assetKeyToText = T.decodeUtf8 . toByteString'
 instance ToSchema AssetKey where
   schema =
     assetKeyToText
-      .= parsedText "AssetKey" (runParser parser . T.encodeUtf8)
+      .= parsedTextWithDoc desc "AssetKey" (runParser parser . T.encodeUtf8)
       & doc' . S.schema . S.example ?~ toJSON ("3-1-47de4580-ae51-4650-acbb-d10c028cb0ac" :: Text)
+    where
+      desc =
+        "S3 asset key for the encrypted icon image with <a href=\"https://github.com/wireapp/\
+        \wire-server/blob/734588952d87e76208d1fa15800a5cfb76d541ee/libs/wire-api/src/Wire/API/\
+        \Asset.hs#L148-L164\">retention information</a>."
```

Comment:
Its not always encrypted, there are assets which are plain like team icon. 
That link will likely get stale, also linking to haskell code is probably not very helpful anyway.